src/wp-admin/includes/ajax-actions.php

```diff
@@ -3970,23 +3970,48 @@ function wp_ajax_crop_image() {
 			/** This filter is documented in wp-admin/includes/class-custom-image-header.php */
 			$cropped = apply_filters( 'wp_create_file_in_uploads', $cropped, $attachment_id ); // For replication.
 
-			$parent_url = wp_get_attachment_url( $attachment_id );
-			$url        = str_replace( wp_basename( $parent_url ), wp_basename( $cropped ), $parent_url );
+			$parent_url      = wp_get_attachment_url( $attachment_id );
+			$parent_basename = wp_basename( $parent_url );
+			$url             = str_replace( $parent_basename, wp_basename( $cropped ), $parent_url );
 
 			$size       = wp_getimagesize( $cropped );
 			$image_type = ( $size ) ? $size['mime'] : 'image/jpeg';
 
+			// Get the original image's post to pre-populate the cropped image.
+			$original_attachment      = get_post( $attachment_id );
+			$sanitized_post_title     = sanitize_file_name( $original_attachment->post_title );
+			$use_original_title       = (
+				( '' !== trim( $original_attachment->post_title ) ) &&
+				/*
+				 * Check if the original image has a title other than the "filename" default,
+				 * meaning the image had a title when originally uploaded or its title was edited.
+				 */
+				( $parent_basename !== $sanitized_post_title ) &&
+				( pathinfo( $parent_basename, PATHINFO_FILENAME ) !== $sanitized_post_title )
+			);
+			$use_original_description = ( '' !== trim( $original_attachment->post_content ) );
+
 			$object = array(
-				'post_title'     => wp_basename( $cropped ),
-				'post_content'   => $url,
+				'post_title'     => $use_original_title ? $original_attachment->post_title : wp_basename( $cropped ),
+				'post_content'   => $use_original_description ? $original_attachment->post_content : $url,
 				'post_mime_type' => $image_type,
 				'guid'           => $url,
 				'context'        => $context,
 			);
 
+			// Copy the image caption attribute (post_excerpt field) from the original image.
+			if ( '' !== trim( $original_attachment->post_excerpt ) ) {
+				$object['post_excerpt'] = $original_attachment->post_excerpt;
+			}
+
 			$attachment_id = wp_insert_attachment( $object, $cropped );
 			$metadata      = wp_generate_attachment_metadata( $attachment_id, $cropped );
 
+			// Copy the image alt text attribute from the original image.
+			if ( '' !== trim( $original_attachment->_wp_attachment_image_alt ) ) {
+				update_post_meta( $attachment_id, '_wp_attachment_image_alt', wp_slash( $original_attachment->_wp_attachment_image_alt ) );
+			}
```

Comment:
Similarly to my above comment, we could integrate this into the associative array passed to `wp_insert_attachment` above, which would colocate it with the other other related data. Other than in my previous comment though, here it would be more about avoiding an extra call to `update_post_meta`.

We can use the `meta_input` key (supported by `wp_insert_post` and thus also by `wp_insert_attachment`) to pass this information, basically like `$object['meta_input'] = array( '_wp_attachment_image_alt' => wp_slash( $original_attachment->_wp_attachment_image_alt ) )`.

Reply:
I absolutely agree. I didn't know `wp_insert_post` supports the `meta_input` key.
Fixed in 2766ec8476d099edd57913a68edfd1071d6aed81
cc @hellofromtonya 
I've implemented the change suggested in @felixarntz`s [comment](https://github.com/WordPress/wordpress-develop/pull/2441#discussion_r834770715) above. I tested it manually and it worked as intended. Unit tests also passed.
But if that will require re-testing all the use cases, please let me know, and I will revert that change.
I also don't want to block this PR from getting merged.